lua/cfc_time/server/storage_options/utils/mysql.lua

```diff
@@ -109,10 +109,23 @@ function storage:PrepareStatements()
           id = ?
     ]]
 
+    local lastSession = string.format( [[
+        SELECT id, joined, departed, duration
+        FROM sessions
+        WHERE
+          user_id = ?
+        AND
+          realm = '%s'
+        ORDER BY
+          departed DESC
+        LIMIT 1
+    ]], realm )
```

Comment:
Gets the latest session information (for this realm!) for the player

Reply:
Previously we just checked if a new user was created during spawn-in to decide if it was their first visit.

This is fine for our case, but if we shared the DB across multiple servers this would be incorrect.

Using this query that actually checks the db for sessions _in this realm_ is the only correct way to know if it's their firs time in this server